### PR TITLE
Configurize

### PR DESCRIPTION
**Pull request by kadamwhite**
This adds a configuration file & instructions for creating your own when installing the app. This step would happen _pre_-build at present.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,43 @@
...
+
+**jsonURI**:
+
+This field holds the URI at which the JSON files output by Ask can be located over HTTP. For local development this will likely be "http://localhost:3000/data/", but in production it will usually be a URL to the location on Amazon S3 to which Ask uploads its JSON.
+
+**formId**:
+
+If you have a local instance of [Coral Ask](https://coralproject.net/products/ask.html) running at, say, localhost:2020, when you're editing the form you wish to use to drive this visualization you will be at a URL similar to this:
+```
+http://localhost:2020/forms/581118892b74a9ceb5c309ab
```
Comment (iros) [inline README.md:31]: What about the hostname and port? Shouldn't we also have that info as part of our form id key?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,43 @@
...
+```
+"581118892b74a9ceb5c309ab" is your formId.
+
+**googleSheetId**:
+
+Editorial text fields in this visualization and the non-user-editable text of the "letter to the president" that is assembled from the responses is managed through [Google Sheets](https://sheets.google.com): create a publicly-visible spreadsheet document using [this format](https://docs.google.com/spreadsheets/d/18yAMk_ydGpPLHTZPrLox7oplvgc-4aswu1arO_IHY9I/edit#gid=904229865). Once your sheet is set up, copy the ID out of the spreadsheet URL; for the URL
```
Comment (iros) [inline README.md:37]: We should tell folks to duplicate that sheet.

**Review by iros — APPROVED** [review]
Looks good, though we should have separated this into two separate PRs - the one merging content.js into api.js and the config file work.

File: README.md (hunk 1 above)
Comment (kadamwhite) [reply README.md:31]: There is no need for this app to talk to ask directly -- that ID governs the name of the output JSON, however, so we combine this and the `jsonURI` variables to determine the name of the file to load.

File: README.md (hunk 2 above)
Comment (kadamwhite) [reply README.md:37]: Good idea, I'll make the change

**Comment by kadamwhite** [comment]
Closing after breaking into #31 and #32

**Pull request closed without merging**